Sources/BraintreePayPal/BTPayPalError.swift

```diff
@@ -70,6 +75,8 @@ enum BTPayPalError: Error, CustomNSError, LocalizedError {
             return "The URL action did not contain a valid URL."
         case .failedToCreateNonce:
             return "Unable to create BTPayPalAccountNonce. Either body did not contain paypalAccounts array or contents could not be parsed."
+        case .webSessionError:
+            return "ASWebAuthenticationSession failed"
```

Comment:
These strings appear in the merchant's application if something goes wrong. So we really try to make these clear and descriptive so the merchant can understand exactly what went wrong!

Reply:
So should I add another error type for webSession failing from start()?

Reply:
I think I will two separate BTPayPalError cases 
case webSessionError(Error)
case authenticationSessionStartFailed. What do you both think?